veniq/dataset_collection/dataflow/graph.dot

```diff
@@ -0,0 +1,128 @@
+digraph {
```

Comment:
delete this file?

Reply:
can we keep it? If we loose it, Egor will be disappointed. Managers need pictures

Reply:
i dont think he meant such a graph